<launch>
  <node pkg="hba" type="visualize_map" name="visualize_map" output="screen">
    <param name="file_path" type="string" value="/home/map/region5-5/"/>
    <param name="downsample_size" type="double" value="0.1"/>
    <param name="marker_size" type="double" value="0.5"/>
    <param name="downsample_size" type="double" value="0.1"/>

    <param name="save_global_map" type="bool" value="true"/>
    
    <param name="pub_step" type="int" value="1"/>
    
    <param name="pcd_name_fill_num" type="int" value="6"/> <!--set 5 for kitti07 and 0 for park-->
  </node>

  <arg name="rviz" default="1"/>
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz_"
    args="-d $(find hba)/rviz_cfg/rosbag.rviz" />
  </group>
</launch>